sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -140,7 +167,7 @@ protected static List<TestEntity> CreateCustomTableEntities(string partitionKeyV
                     GuidTypeProperty = new Guid($"0d391d16-97f1-4b9a-be68-4cc871f9{n:D4}"),
                     BinaryTypeProperty = new byte[] { 0x01, 0x02, 0x03, 0x04, 0x05 },
                     Int64TypeProperty = long.Parse(number),
-                    DoubleTypeProperty = (double)n,
+                    DoubleTypeProperty = double.Parse($"{number}.0"),
```

Comment:
Interesting change. What is the reason for it?

Reply:
It is to validate that serialization ignores the trailing `.0`. See #13552